MantidQt/MantidWidgets/src/InstrumentView/InstrumentWidgetPickTab.cpp

```diff
@@ -961,6 +983,60 @@ void ComponentInfoController::displayComparePeaksInfo(
   m_selectionInfoDisplay->setText(QString::fromStdString(text.str()));
 }
 
+void ComponentInfoController::displayAlignPeaksInfo(
+    const std::vector<Mantid::Kernel::V3D> &planePeaks,
+    const Mantid::Geometry::IPeak *peak) {
+  if (planePeaks.size() < 3)
+    return;
+
+  auto pos1 = planePeaks[0];
+  auto pos2 = planePeaks[1];
+  auto pos3 = planePeaks[2];
```

Comment:
I was actually thinking of doing it earlier on in ProjectionSurface, so that the event does not fire until you have uniquely defined points.